src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.impl.response.UserResponseMapper;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService service;
+    private final UserResponseMapper userResponseMapper;
+
+    public AuthenticationController(AuthenticationService service,
+                                    UserResponseMapper userResponseMapper) {
+        this.service = service;
+        this.userResponseMapper = userResponseMapper;
+    }
+
+    @PostMapping("/register")
+    public UserResponseDto register(@RequestBody UserRequestDto requestDto) {
+        return userResponseMapper.toDto(service
+               .register(requestDto.getEmail(), requestDto.getPassword()));
```

Comment:
```suggestion
        User user = authenticationService .register(requestDto.getEmail(), requestDto.getPassword()));
         return userResponseMapper.toDto(user);
```
let's increase readability :)